docs/DeveloperGuide_Winery.md

```diff
@@ -0,0 +1,622 @@
+---
+{}
+---
+---
+---
+<center>
+
+![alt text](graphics/Figure00-Cloudcycle.png "Cloudcyle")
```

Comment:
Do not use "alt text", do the real description. Move `"Cloudcycle"`  to "alt text" and write it "CloudCycle".